vcpkg/build_iroha_deps.sh

```diff
@@ -8,5 +8,12 @@ git clone https://github.com/microsoft/vcpkg $vcpkg_path
 git -C $vcpkg_path checkout $(cat "$iroha_vcpkg_path"/VCPKG_COMMIT_SHA)
 for i in "$iroha_vcpkg_path"/patches/*.patch; do git -C $vcpkg_path apply --ignore-whitespace $i; done;
 $vcpkg_path/bootstrap-vcpkg.sh
+
+ARCH="$(uname -m)"
+
+if [ "$ARCH" = "armv7l" -o "$ARCH" = "aarch64" -o "$ARCH" = "s390x" -o "$ARCH" = "ppc64le" ]; then
+    export VCPKG_FORCE_SYSTEM_BINARIES=1
```

Comment:
it is impacting commands in the script, but calling (according to documentation:
https://iroha.readthedocs.io/en/master/build/index.html#linux-and-macos)
requires the flag again:
```
VCPKG_FORCE_SYSTEM_BINARIES=1 vcpkg/vcpkg integrate install
```
I'll describe its in documentation:)